packages/stacks-docs/product/components/tags.html

```diff
@@ -37,15 +43,15 @@
 {% highlight html %}
 <div class="d-flex g4">
     <a class="s-tag" href="#">jquery</a>
-    <span class="s-tag">javascript <button class="s-tag--dismiss">@Svg.ClearSm</button></span>
-    <a class="s-tag" href="#"><img class="s-tag--sponsor" src="https://i.stack.imgur.com/tKsDb.png" width="16" height="18" alt="Google Android"> android</a>
+    <span class="s-tag" tabindex="0">javascript <button class="s-tag--dismiss"><span class="v-visible-sr">Dismiss</span>@Svg.ClearSm</button></span>
```

Comment:
NIT: The screen reader text should give an idea of what the focused element does. `Dismiss` is good, but I think something like `Dismiss tag` or maybe even `Dismiss javascript` would be provide a good example.

```suggestion
    <span class="s-tag" tabindex="0">javascript <button class="s-tag--dismiss"><span class="v-visible-sr">Dismiss</span>@Svg.ClearSm</button></span>
```

Reply:
I went with your suggestion as it also resolved the issue I was trying to fix with `tabindex="0"`!